guides/create-graphql-mutation.md

```diff
@@ -0,0 +1,234 @@
+## At a glance
+
+You're working on an implementation for Leader of the Pack, an outdoor equipment retailer. In the GraphQL query guide we described data we added that contains whether we can ship a category of products to a particular zip. But now we need a way to update that information. For this we will create a GraphQL mutation
+
+## What you need 
+
+You need to understand what key you will use to query the data, and what the payload of the data is.
+
+## Identify which plugin owns the mutation
+
+When adding a new mutation, the first step is to decide which plugin should own it. This is usually obvious, but not always. You should think about whether any other plugins or services will need to call your mutation.
+
+## Define the mutation in the schema
+
+- If it doesn't already exist, create `schemas` folder in the plugin, and add an `index.js` file there.
```

Comment:
Similar comment to the other. I would drop the bullets and try to consolidate a little, like:

If it doesn't already exist, create schemas folder in the plugin and add an index.js file there. Then check to see if there is a schema.graphql file in your schemas directory within the plugin. If there isn't, create that file now.

Import the GraphQL file...